api/http-request-practice.md

```diff
@@ -163,6 +161,6 @@ Sync up with your pair or another apprentice who is available and go through the
 ### Supplemental Materials
 
 - [Axios](https://github.com/axios/axios) - a popular HTTP client library for Node
-- [freeCodeCamp.org](https://www.freecodecamp.org/news/a-practical-es6-guide-on-how-to-perform-http-requests-using-the-fetch-api-594c3d91a547/)
+- [freeCodeCamp.org](https://www.freecodecamp.org/news/a-practical-es6-guide-on-how-to-perform-http-requests-using-the-fetch-api-594c3d91a547/) - consists of free tutorials to help in understanding programming tools and concepts
```

Comment:
I think you could be more specific about the content that user will see after going to the respective link.